charts/posthog/values.yaml

```diff
@@ -481,7 +481,14 @@ kafka:
     # - Enable data persistence using PVC.
     enabled: true
     # -- PVC Storage Request for Kafka data volume.
-    size: 20Gi
+    size: 80Gi
+  # -- A size-based retention policy for logs
+  # -- Should be less than kafka.persistence.size, minimum 1GB
+  logRetentionBytes: _2_333_333_333
+  # -- The minimum age of a log file to be eligible for deletion due to age
+  logRetentionHours: 24
+  # -- The default number of partitions per topic	
+  numPartitions: 128
```

Comment:
This is way too many for anyone except people processing billions of events. We should be tuning these defaults for the average-sized companies.